Docker/docker-compose.yml

```
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  MCC:
+    image: 'minecraft-console-client:latest'
+    container_name: 'MinecraftConsoleClient'
+    # You could also comment out the volumes section if you don't want to mount the volume. That is up to you. If you do so it's harder to edit the .ini file if thats something you want to do.
+    volumes:
+    - '<PATH_ON_YOUR_MACHINE_TO_MOUNT>:/opt/data'
```

How about filling relative path here?
```suggestion
    - './data:/opt/data'
```

---

This would work as well. I used it in my example in the user manual as well. The only point with this is that a folder is created in the user's cloned repository. But it is fine for me to do it this way. Shall I change it and maybe clean up the user manual as well? Then the user has to change nothing by default.

---

Yes, please.